static/js/apps/eval_retrieval_generation/util.ts

```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
-const httpPattern = /https:\/\/[^\s]+/g;
+import _ from "lodash";
 
-const longSpaces = "&nbsp;&nbsp;&nbsp;&nbsp;";
+const HTTP_PATTERN = /https:\/\/[^\s]+/g;
+const LONG_SPACES = "&nbsp;&nbsp;&nbsp;&nbsp;";
+const TABLE_DIVIDER_PATTERN = /[--][-]+/g;
+const TABLE_HEADER_TEXT_PATTERN = /[\w'ô[\]ãéí\s]+/g;
```

Comment:
why the pattern is like this..?

Reply:
because there are country names in the header and some country names have those symbols like "Côte d'Ivoire"

Reply:
i see, worth to have a comment.

Reply:
done